<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActimetryConfigWidget</class>
 <widget class="QWidget" name="ActimetryConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActimetryConfigWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabMain">
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <widget class="QWidget" name="tabContext">
      <attribute name="icon">
       <iconset resource="../../../resources/TeraClient.qrc">
        <normaloff>:/icons/details.png</normaloff>:/icons/details.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Algorithmes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="lblWarning">
         <property name="text">
          <string>Aucun type de séance lié au service &quot;Actimetry&quot; associé à ce projet. Veuillez ajouter un type de séance approprié.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameContextUpdater">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="framePlaylistControls">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="lblSelectAlgorithm">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Sélectionnez un algorithme</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboAlgorithms">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>80</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblDescription">
               <property name="text">
                <string>Description</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="textEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="lblParameters">
               <property name="text">
                <string>Paramètres</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="textEditParameters"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnRun">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>3</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>46</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Démarrer</string>
               </property>
               <property name="icon">
                <iconset resource="../../../resources/TeraClient.qrc">
                 <normaloff>:/controls/branch_closed.png</normaloff>:/controls/branch_closed.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ResultMessageWidget" name="wdgMessages" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResultMessageWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ResultMessageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/TeraClient.qrc"/>
 </resources>
 <connections/>
</ui>
